### ISSUE TITLE
Possibly redundant functionality

### ISSUE THREAD

**Issue by butzist** [plotly/plotly.rs#123]
A few days ago I wrote and released a [crate](https://crates.io/crates/yew-plotly) featuring a yew component wrapper that uses your library. Now I noticed you also just recently extended the wasm support and added examples for yew.

I wonder if it would make sense integrating this crate into your repo, or if we could move both repos to a joint github org?

**Comment by mfreeborn** [comment]
My feelings with regards to Wasm support is to provide the framework-agnostic basic building blocks so that other libraries can be build on top of it.

The problem with integrating e.g. `Yew` directly into the `plotly.rs` library is that you then have to wonder, should we include a wrapper for `sycamore` and `seed` components? What about as-yet-unwritten frontend frameworks? What if the community moves away from `Yew` - should we still support a `Yew` component wrapper?

So I would leave the level of Wasm support as it is at the moment i.e. just provides some convenient bindings to `plotly.js` and I'd leave it to the user to handle how they integrate that with their frontend of choice.

In terms of examples, I'd definitely like more examples of usage within other frontend frameworks. The only reason that the only example is for `Yew` is because that is the framework I am familiar with personally.

I think there is space in the ecosystem for separate libraries such as yours which specialise for different frontends.

**Comment by butzist** [comment]
Makes sense to me. Then the maintenance of the yew-specific support could be moved to my crate and I set up some CI tests on my side to assure future compatibility.
How do you suggest to structure the docs? I could create a PR integrating usage examples of the `yew-plotly` crate in your readme/book.

**Comment by andrei-ng** [comment]
@butzist , would you still be interested to add and example to `examples` and to the mdBook regarding usage of your `yew-plotly` crate?

I am looking through old open issues and deciding which ones to close. Let me know if you still want to do something on your end, otherwise I will close the issue.
